### ISSUE TITLE
Endless loop for large persistence lengths

### ISSUE THREAD

**Issue by jan-stevens** [marrink-lab/polyply_1.0#185]
## Expected behavior
Using the build files as input, Polyply allows the user to specify a desired persistence length for generated molecules. When large persistence lengths (`lp > end-to-end distance`) are passed by the user, a rod-like molecule should be produced or if not possible an error should be thrown. 

## Actual behavior
Currently, if the user defined persistence length is larger then the specified molecules end-to-end distance, the code can't produce molecule coordinates. What happens is that the code enters an endless loop until the the user terminates the process manually.

## Code to reproduce the behavior

The command line inputs are
```
polyply gen_params -lib martini3 -o PEO10.itp -name PEO -seq PEO:10
polyply gen_coords -p PEO10.top -o PEO10.gro -name PEO -dens 1000 -b build_file
```

Here `gen_coords` asks as input the `PEO.top` topology file for our polymer:
```
#include "martini_v3.0.0.itp"
#include "PEO10.itp"
[ system ]
; name
Melt of PEO 10

[ molecules ]
; name  number
PEO 10
```

And finally the `build_file` that specifies the large persistence length required to reproduce the behavior
```
[ molecule ]
PEO 0 1
[ persistence_length ]
WCM 10 0 9
```

**Comment by fgrunewald** [comment]
@jan-stevens your code should issue a Warning with the settings that you post above. That warning tells you that under these settings your longest end-to-end distance is shorter than your box vector. This means that the longest end-to-end distance has to align along the diagonal of the box. I think that's why this is not converging. So I don't see that as a problem really.

**Comment by fgrunewald** [comment]
Scratch the last part. I don't find an issue here if the box is large enough (i.e. the box needs to be sufficiently much larger than the largest end-to-end distance.)

**Comment by fgrunewald** [comment]
solved in #204
